MAUI/DataGrid/summaries.md

```diff
@@ -65,6 +65,39 @@ The following screenshot shows the outcome of the previous code:
 
 ![DataGrid with formatting in caption summary row](Images\caption-summary\maui-datagrid-formatting-show-summary-in-row.png)
 
+### Display a custom name in place of the column name in the group caption summary text
+
+The column name in the group caption summary text can be customized using the DisplayName property.
+
+If the DisplayName property is set, the specified display name will appear in the group caption summary text. If it is null, the column name will be displayed instead.
```

Comment:
Provide backtick (`) to the DisplayName property.

DisplayName => `DisplayName`